### DISCUSSION TITLE
Window gets tabbed when move to workspace with a tabbed window

### DISCUSSION THREAD

**Discussion by medwatt** [i3/i3#4918] [Q&A]
I have some applications that I want to always open in tabbed mode. The way to do it seems to be `for_window [class="okular"] layout tabbed`. This is fine except when I move a window from one workspace to the workspace where the application with a tabbed layout is, the application also gets tabbed. Is it possible to change this behavior?

**Comment by Airblader** [comment]
No, not through configuration. You will need to write a script using the IPC that handles this.
